nosedrop grant

transmissions11 opened the pull request.

test9955667 committed:
added my WIP to fork

Adding my 50% edited lens docs to the fork so its up there. Hopefully I did this right im known to fuck up VC

test9955667 committed:
testing solidity markdown header

test9955667 committed:
added bullets to dropdown

test9955667 committed:
Merge branch 'Rari-Capital:main' into main

test9955667 committed:
missing </details>

test9955667 committed:
added missing bullet

test9955667 committed:
fixing formatting issues

test9955667 committed:
added function headers

test9955667 committed:
finishing liquidator

test9955667 committed:
Final updates before 1st review

test9955667 committed:
added liq header

jbenus commented:
Linking to the 3 <h5> return values, pool asset and user is not working.

test9955667 commented:
linking to the 3 "<h5>" return values is not working

test9955667 committed:
test

test9955667 committed:
test 2

test9955667 commented:
Alright last annoying thing is getting links to html headers (hx notation) to work. not super difficult but its 7 am and thats not the morning

test9955667 committed:
attempt to fix links

test9955667 committed:
Attempt 3 to link headers

test9955667 committed:
attempt 3 to get links working

test9955667 committed:
attempt *5 at linking

test9955667 committed:
style updates (read)

de-structured solidity returns for copypsata, fixed capitalization error of structs, "verticalized"  long lines

test9955667 committed:
removed WIP edits
